modules/core/src/v2/coins/trx.ts

```diff
@@ -192,4 +205,47 @@ export class Trx extends BaseCoin {
 
     return Buffer.from(sig, 'hex');
   }
+
+  /**
+   * Explain a Tron transaction from txHex
+   * @param params
+   * @param callback
+   */
+  explainTransaction(
+    params: ExplainTransactionOptions,
+    callback?: NodeCallback<TransactionExplanation>
+  ): Bluebird<TransactionExplanation> {
+    return co<TransactionExplanation>(function*() {
+      const txHex = params.txHex || (params.transaction && params.transaction.txHex);
+      if (!txHex || !Tron.safeTronHexString(txHex)) {
+        throw new Error('missing valid txHex string');
+      }
+      if (!params.fee) {
+        throw new Error('missing fee');
+      }
+      if (!params.txID) {
+        throw new Error('missing txID');
+      }
+      const rawContract = Tron.getRawTransferContract(txHex);
+      const transfer = Tron.decodeTransferContract(rawContract.contractList[0].parameter.value);
```

Comment:
Change to account lib util decodeTransferContract: https://github.com/BitGo/bitgo-account-lib/pull/5/files#diff-77e5edd96bbcbf039a06a7be97358a2aR149 trx/utils.ts line 149